chatgpt/api.py

```diff
@@ -110,7 +114,7 @@ def send_message(self, message: str) -> Response:
             parent_msg_id=self._parent_message_id,
         )
         response = self.post(
-            self._CONV_URL, headers=self._chatgpt_headers, content=data
+            self._CONV_URL, headers=self._chatgpt_headers, data=data
```

Comment:
This causes `DeprecationWarning` from `httpx` package. It's recommended to use `content` argument (I also changed in 0.2.1, see: https://github.com/mbroton/chatgpt-api/pull/9/files).

Reply:
Fixed it.

Reply:
Are you going to push it? 😄

Reply:
Just did it ! 🌝